__tests__/Unit/Components/Tasks/Suggestions.test.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { fireEvent, screen, waitFor } from '@testing-library/react';
+import { renderWithRouter } from '@/test_utils/createMockRouter';
+import { Provider } from 'react-redux';
+import { store } from '@/app/store';
+import { setupServer } from 'msw/node';
+
+import usersHandler from '../../../../__mocks__/handlers/users.handler';
+import Suggestions from '@/components/tasks/SuggestionBox/Suggestions';
+
+import handlers from '../../../../__mocks__/handlers';
+const server = setupServer(...handlers);
```

Comment:
why spaces between import...it's not required and also there should be space after line 11.